database/changesets/20210422_ContinuousWorkloadTable.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext
+                        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd
+                        http://www.liquibase.org/xml/ns/dbchangelog
+                        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+    <changeSet author="chengche@broadinstitute.org"
+               dbms="postgresql"
+               id="20210422"
+               logicalFilePath="20210422_ContinuousWorkloadTable.xml"
+               runInTransaction="false">
+        <sql>
+            CREATE TABLE ContinuousWorkload (
+                id              BIGINT GENERATED BY DEFAULT AS IDENTITY NOT NULL,          -- primary key
+                source_type     SOURCE NOT NULL,                                           -- type of this source
+                source_item     TEXT NOT NULL,                                             -- the identifier of the source table
+                executor_type   EXECUTOR NOT NULL,                                         -- type of this executor
+                executor_item   TEXT NOT NULL,                                             -- the identifier of the executor table
+                sink_type SINK  NOT NULL,                                                  -- type of this sink
```

Comment:
Nit: spacing to align with other column types for this and `sink_item`.

Reply:
nice catch! will fix